smart-contracts/contracts/AtlantisToken.sol

```diff
@@ -17,8 +17,15 @@ contract Atlantis is ERC20, Ownable {
         }
         _;
     } 
+    // constructor mints 1 million Atlantis token
+    constructor() ERC20("Atlantis", "ALT") {
+        _mint(msg.sender, 1000000 * 10 ** 18);
```

Comment:
Hello boss, please I will be declining this. We do not need to mint when deploying the contract